plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/tools/project/ProjectExportWizard.java

```diff
@@ -119,6 +119,14 @@ private void exportProjects(DBRProgressMonitor monitor, final ProjectExportData
 
         String archiveName = exportData.getArchiveFileName() + ExportConstants.ARCHIVE_FILE_EXT;
         File archiveFile = new File(exportData.getOutputFolder(), archiveName);
+        
+        if (archiveFile.exists()) {
+        	boolean overwrite = DBWorkbench.getPlatformUI().confirmAction(CoreMessages.dialog_project_export_wizard_file_overwrite_window_title, 
+        			NLS.bind(CoreMessages.dialog_project_export_wizard_file_overwrite_confirm, archiveName), true);
+        	
+        	if (!overwrite) return;
+        }
```

Comment:
Please don't mix tabs and spaces. We use 4 space indents. It's also good to break long expressions into several lines:

```suggestion
        if (archiveFile.exists()) {
            boolean overwrite = DBWorkbench.getPlatformUI().confirmAction(
                CoreMessages.dialog_project_export_wizard_file_overwrite_window_title,
                NLS.bind(CoreMessages.dialog_project_export_wizard_file_overwrite_confirm, archiveName),
                true
            );

            if (!overwrite) {
                return;
            }
        }
```